src/main/resources/META-INF/resources/primefaces/tabview/tabview.js

```diff
@@ -58,6 +58,7 @@
  * @prop {boolean} cfg.scrollable When enabled, tab headers can be scrolled horizontally instead of wrapping.
  * @prop {number} cfg.selected The currently selected tab.
  * @prop {number} cfg.tabindex Position of the element in the tabbing order.
+ * @prop {boolean} cfg.multiViewState Whether to keep TabView state across views.
```

Comment:
@blutorange : Is it right this way? IntelliJ had not been happy because it thought cfg does not contain multiViewState.

Reply:
@christophs78 Looks good to me. The IDE support for the JSDocs is not perfect, we would probably have to switch to ES6 classes and/or TypeScript for that. That said, IntelliJ probably reads the current PrimeFaces.d.ts file, where this property does not exist yet. That file is generated automatically via `mvn clean install -P release` I guess we should run that occasionally and commit the generated files.

Also, once you open a PR, the Travis build check whether the JSDocs are valid. You can also check that locally by running `mvn clean install -P jsdoc` (see also the readme https://github.com/primefaces/primefaces/blob/master/src/main/type-definitions/README.md#build)